### ISSUE TITLE
PR Call Node Script Event

### ISSUE THREAD

**Issue by ellogwen** [coppolaemilio/dialogic#128]
TLDR;
https://github.com/coppolaemilio/dialogic/pull/130

Hi guys, I hope you are not mad at me, but I created a new event node for time timeline to call arbitrary node methods, because I needed it badly. I could decide if I just have to create a workaround or contribute instead. I do not want to take over your project, but maybe you find this handy and are considering to add it? It was not that hard to do, so I am not mad if you decline it.

Summary:
I created a new event node call "Call Node". You can setup the node to call with a default Godot NodePath and a method name. In addition you can add arguments as well. The Timeline will execute those methods and even wait for completion, if the method in question is async and/or yielding.

I provided an example video that shows the new feature in question (Sorry for the odd video aspect ratio. OBS kind of dislikes me today):

https://streamable.com/s79dsk

Here is an example of the timeline:

![image](https://user-images.githubusercontent.com/22261975/113137458-dd0c9780-9224-11eb-92fc-e2b5437b4153.png)
![image](https://user-images.githubusercontent.com/22261975/113137520-ee55a400-9224-11eb-9533-7f93ee4de583.png)

I tried to follow your conventions as much as possible, and tried to not alter any existing stuff. However, I needed to change a small piece of the dialog_node.gd in order to make yield-waiting work. This change should not have any sideeffects.

Please let me know what you think and if I need to change something. 

Edit: here is the PR
https://github.com/coppolaemilio/dialogic/pull/130

**Comment by coppolaemilio** [comment]
This looks really good! I'll review it tonight and let you know, but I really like the idea of exposing more coding to the timelines if needed.

**Comment by arnaudvergnet** [comment]
@coppolaemilio you can close this issue now that the pr is merged or is there something left to do?
